website/docs/getting-started/using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md

```diff
@@ -0,0 +1,68 @@
+---
+id: using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md
+title: Using the `podman-mac-helper` tool on macOS
+description: Using the `podman-mac-helper` tool can make it easier to migrate from Docker to Podman on macOS, as it allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+tags: [podman-desktop, getting-started, podman-mac-helper, macOS]
+keywords: [podman desktop, podman, containers]
+---
+
+## Using the `podman-mac-helper` tool to migrate from Docker to Podman on macOS
+
+The `podman-mac-helper` tool provides a compatibility layer that allows you to use most Docker commands with Podman on macOS.
+The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket location.
+
+Using the `podman-mac-helper` tool can make it easier to migrate from Docker to Podman on macOS.
+It allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+
+#### Prerequisites
+
+* macOS
+* [Podman](../Installation/macos-install)
+* [Homebrew](https://brew.sh/)
+
+#### Procedure
+
+1. Save your existing Docker containers by running the command:
+
+    ```
+    $ docker save <your_container> > <your_container_archive>.tar
+    ```
+
+2. Disable or uninstall the Docker service.
```

Comment:
maybe we should indicate what to do? 
I believe by "disabling" we mean "shutting down" Docker service.

Reply:
I am not sure of this step, because I can't test it properly.

Reply:
https://docs.docker.com/desktop/use-desktop/pause/ maybe?

Reply:
Better?